packages/console/src/components/AppContent/components/Sidebar/components/Item/index.module.scss

```diff
@@ -19,7 +19,7 @@
     flex: 0 0 _.unit(5);
 
     svg {
-      fill: var(--color-neutral-variant-50);
+      color: var(--color-neutral-variant-50);
```

Comment:
I think this one can be omitted, as the color will be inherited from parent `.row`